[DOC] Procedure for upgrading AMQ Streams on RHEL (#40)
rh-messaging/amq-streams-docs: books/assembly-overview.adoc

Zookeeper:: Service for highly reliable distributed coordination.
Kafka Broker:: Messaging broker responsible for delivering records from producing clients to consuming clients.
Kafka Connect:: A toolkit for streaming data between Kafka brokers and other systems using _Connector_ plugins.
Kafka Consumer and Producer APIs:: Java based APIs for producing and consuming messages to and from Kafka brokers.
Kafka Streams API:: API for writing _stream processor_ applications.

> The Kafka broker is the central point connecting all these components. 

Reviewer: I know it's talking abstractly about "the Kafka broker", but I nonetheless think this is a little misleading because it's talking about a singular broker, but in reality it's a cluster. 

```suggestion
A cluster of Kafka brokers is the hub connecting all these components. 
```
